bower.json

```diff
@@ -25,5 +25,8 @@
     "angular": "~1.2.18",
     "angular-mocks": "~1.2.18",
     "angular-ui-tinymce": "latest"
+  },
+  "resolutions": {
```

Comment:
is this really necessary ?  pretty sure it would have been in there already if necessary